Transformer/SqlToBddTransformer.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace OpenOrchestra\Transformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * Class SqlToBddTransformer
+ */
+class SqlToBddTransformer implements DataTransformerInterface
```

Comment:
I'd call this more BooleanLogicToBddTransformer or something like this, as it sounds weird to talk about Sql in a MongoBundle